From 879b60c272265dc87608f7baa75e6811711747d9 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Fri, 11 May 2018 14:10:00 +0800
Subject: [PATCH 3772/5242] MLK-18281 clk: imx8mm: correct mistypes for
 'sys_pll1_800m'

commit  8ed73d1a916828b204b2101b8acd253011c323f9 from
https://source.codeaurora.org/external/imx/linux-imx.git

Two 'sys_pll1_800m's are mistyped to 'sys1_pll_800m'.
So correct them.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/clk/imx/clk-imx8mm.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/clk/imx/clk-imx8mm.c b/drivers/clk/imx/clk-imx8mm.c
index f458e90..12d928c 100644
--- a/drivers/clk/imx/clk-imx8mm.c
+++ b/drivers/clk/imx/clk-imx8mm.c
@@ -165,7 +165,7 @@ enum {
 static const char *imx8mm_disp_axi_sels[] = {"osc_24m", "sys_pll2_1000m", "sys_pll1_800m", "sys_pll3_out",
 					     "sys_pll1_40m", "audio_pll2_out", "clk_ext1", "clk_ext4", };
 
-static const char *imx8mm_disp_apb_sels[] = {"osc_24m", "sys_pll2_125m", "sys1_pll_800m", "sys_pll3_out",
+static const char *imx8mm_disp_apb_sels[] = {"osc_24m", "sys_pll2_125m", "sys_pll1_800m", "sys_pll3_out",
 					     "sys1_pll_40m", "audio_pll2_out", "clk_ext1", "clk_ext3", };
 
 static const char *imx8mm_disp_rtrm_sels[] = {"osc_24m", "sys_pll1_800m", "sys_pll2_200m", "sys_pll2_1000m",
@@ -261,7 +261,7 @@ enum {
 static const char *imx8mm_nand_sels[] = {"osc_24m", "sys_pll2_500m", "audio_pll1_out", "sys_pll1_400m",
 					 "audio_pll2_out", "sys_pll3_out", "sys_pll2_250m", "video_pll1_out", };
 
-static const char *imx8mm_qspi_sels[] = {"osc_24m", "sys1_pll_400m", "sys1_pll_800m", "sys2_pll_500m",
+static const char *imx8mm_qspi_sels[] = {"osc_24m", "sys1_pll_400m", "sys_pll1_800m", "sys2_pll_500m",
 					 "audio_pll2_out", "sys1_pll_266m", "sys3_pll2_out", "sys1_pll_100m", };
 
 static const char *imx8mm_usdhc1_sels[] = {"osc_24m", "sys_pll1_400m", "sys_pll1_800m", "sys_pll2_500m",
-- 
1.7.9.5

